### DISCUSSION TITLE
Treat as Urgent please.

### DISCUSSION THREAD

**Discussion by AccoladesIO** [vercel/community#1810] [Help]
My Images loads well in my localhost but doesn't load when i deploy to vercel. What might be the issue

**Comment by swarnava** [comment]
Can you share a [minimal repository](https://stackoverflow.com/help/minimal-reproducible-example) which we can take a closer look?

**Comment by AccoladesIO** [comment]
[link to my repo](https://github.com/AccoladesIO/Coin-Price-Tracker)

**Comment by amyegan** [comment]
Thanks for sharing your repo, @AccoladesIO . I'm not able to reproduce the image problem. If you're still seeing this problem, can you share screenshots or describe exactly the problem we should see? I might be looking at the wrong thing

**Reply by AccoladesIO** [reply]
Whenever I deploy my repo to my Localhost the images show properly. But on deploying to vercel, the image didn't show. I shared my repo so you could help me check if there's something I'm not doing well.

**Reply by AccoladesIO** [reply]
![Screenshot_20230320-170139_GitHub.jpg](https://user-images.githubusercontent.com/73618922/226399022-59e248c3-cc82-4d8d-b751-3788e671b233.jpg)

![Screenshot_20230320-170207_GitHub.jpg](https://user-images.githubusercontent.com/73618922/226399023-440be96d-f1b4-4a1e-8b30-60ee29f3c54f.jpg)

**Reply by amyegan** [reply]
I see the images on both local development and a preview deployment. Can you share an example deployment and/or screenshots of the differences you see between local and deployed versions of the site?
